### PR TITLE
docs: DOC-1177

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR fixes an issue where we did not correctly call out how to deploy to an external registry. 

## Changed Pages

💻 [Add Preview URL for Page]()

## Jira Tickets

🎫 [Jira Ticket](DOC-1177)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._

### CODE REVIEW

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md [hunk 1]
```diff
@@ -114,7 +114,18 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
    [Upload Cluster Images to External Registry with Palette Edge CLI](./upload-images-to-registry.md) to learn how to
    use the Palette Edge CLI to upload all images in a cluster profile to an external registry.
 
-9. Follow the [Create Cluster Definition](../site-installation/cluster-deployment.md) guide and deploy your cluster.
+9. In the Kubernetes layer of your cluster profile, remove `AlwaysPullImages` from
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins` and add it to
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`.
```
Comment (rishi-anand) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md:119]: @lennessyy We need to just remove `AlwaysPullImages` and don't have to add it anywhere. So we should remove "add it to
   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`"

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md [hunk 2]
```diff
@@ -114,7 +114,18 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
...
+
+   The resulting layer configuration should look like the following.
+
+   ```yaml
+   disable-admission-plugins: AlwaysPullImages,AlwaysAdmit
```
Comment (rishi-anand) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md:124]: @brathina-spectro Do you have to explicitly add `AlwaysPullImages` in `disable-admission-plugins`?

From what I recollect, we just used to remove `AlwaysPullImages` from `enable-admission-plugins`?

Comment (brathina-spectro) [reply docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md:124]: `AlwaysPullImages` admission plugin should be removed from enable-admission-plugin. Not changes are needed in disable-admission. You can refer what we've documented already for [airgap use case](https://docs.spectrocloud.com/integrations/harbor-edge/#prerequisites)

**Comment by rishi-anand** [comment]
@lennessyy place where we have below line, we can mention that kubernetes core images like api-server, etcd, kube-controller-manager are baked in the provider image itself and these images will be loaded into containerd and won't be downloaded

_The Palette agent will automatically download the image using the tag 10.10.254.254:8000/spectro-images/quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1 instead of looking for the image in the original registry._

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md [hunk 3]
```diff
@@ -115,13 +117,12 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
...
 
 9. In the Kubernetes layer of your cluster profile, remove `AlwaysPullImages` from
-   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins` and add it to
    `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`.
 
    The resulting layer configuration should look like the following.
 
    ```yaml
-   disable-admission-plugins: AlwaysPullImages,AlwaysAdmit
+   disable-admission-plugins: AlwaysAdmit
```
Comment (rishi-anand) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md:125]: @lennessyy May be we can skip any change in `disable-admission-plugins` section. It's not there in released docs https://docs.spectrocloud.com/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry/ and for above external registry's changes, we don't need to make any change in `disable-admission-plugins`.

**Review by rishi-anand — APPROVED** [review]

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-custom-registries.md [hunk 4]
```diff
@@ -37,12 +37,15 @@ After the provider images are downloaded, the Palette agent proceeds to download
...
-tag `10.10.254.254:8000/spectro-images/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1` instead of looking for the image in the
-`quay.io/kairos` registry.
+When an external registry is provided in the user data, the Palette agent will prepend the URL of the external registry
+to the URL of every image. For example, if your OS pack specified that the provider images be downloaded from
+`quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1`, but in your user data, you have specified an external registry
+`10.10.254.254:8000/spectro-images`. The Palette agent will automatically download the image using the tag
+`10.10.254.254:8000/spectro-images/quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1` instead of looking for the
+image in the original registry.
+
+The provider image also includes core Kubernetes images such as for images api-server, etcd, kube-controller-manager,
```
Comment (caroldelwing) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-custom-registries.md:47]:
```suggestion
The provider image also includes core Kubernetes images such as images for api-server, etcd, and kube-controller-manager,
```
maybe?

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-custom-registries.md [hunk 5]
```diff
@@ -37,12 +37,15 @@ After the provider images are downloaded, the Palette agent proceeds to download
...
-When an external registry is provided in the user data, the Palette agent will replace the registry URL of every image
-used by the cluster with the external registry. For example, if your OS pack specified that the provider images be
-downloaded from `quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1`, but in your user data, you have specified an
-external registry `10.10.254.254:8000/spectro-images`. The Palette agent will automatically download the image using the
-tag `10.10.254.254:8000/spectro-images/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1` instead of looking for the image in the
-`quay.io/kairos` registry.
+When an external registry is provided in the user data, the Palette agent will prepend the URL of the external registry
+to the URL of every image. For example, if your OS pack specified that the provider images be downloaded from
+`quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1`, but in your user data, you have specified an external registry
+`10.10.254.254:8000/spectro-images`. The Palette agent will automatically download the image using the tag
```
Comment (caroldelwing) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-custom-registries.md:43]:
```suggestion
as `10.10.254.254:8000/spectro-images`, the Palette agent will automatically download the image using the tag
```

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md [hunk 6]
```diff
@@ -13,12 +13,15 @@ for the registry in the user data used to build your Edge Installer ISO.
...
 external image registry. For example, if your OS pack specified that the provider images be downloaded from
 `quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1`, but in your user data, you have specified an external registry
-`10.10.254.254:8000/spectro-images`. The Palette agent will automatically download the image using the tag
-`10.10.254.254:8000/spectro-images/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1` instead of looking for the image in the
-`quay.io/kairos` registry.
+`10.10.254.254:8000/spectro-images/`. The Palette agent will automatically download the image using the tag
+`10.10.254.254:8000/spectro-images/quay.io/kairos/core-ubuntu-20-lts-rke2:v1.25.2-rke2r1` instead of looking for the
+image in the original registry.
+
+The provider image also includes core Kubernetes images such as for images api-server, etcd, kube-controller-manager,
```
Comment (caroldelwing) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md:23]:
```suggestion
The provider image also includes core Kubernetes images such as images for api-server, etcd, and kube-controller-manager,
```

**Review by caroldelwing — APPROVED** [review]
:rocket: tiny suggestions

**Pull request merged**